Makefile.buildx.mk

```diff
@@ -21,6 +21,23 @@ ifeq (${TAG}, )
   export TAG=ci
 endif
 
+# default list of platforms for which multiarch image is built
+ifeq (${PLATFORMS}, )
+	export PLATFORMS="linux/amd64,linux/arm64,linux/arm/v7,linux/ppc64le"
+endif
+
+# if IMG_RESULT is unspecified, by default the image will be pushed to registry
+ifeq (${IMG_RESULT}, load)
+	export PUSH_ARG="--load"
+    # if load is specified, image will be built only for the build machine architecture.
+    export PLATFORMS=${XC_OS}/${XC_ARCH}
```

Comment:
Has this been tested? Scenarios that this might not work are in cases where you have a strict ARM type that is required, EG. `linux/arm/v7` - I don't think this would build this variable correctly.

Reply:
Will change to [`local`](https://github.com/docker/buildx/blob/f3111bcbef8ce7e3933711358419fa18294b3daf/util/platformutil/parse.go#L34). It will get the host machine platform and compile for it.  It automatically takes care of the [cpu variant](https://github.com/containerd/containerd/blob/422c401a4866fae8543c7f771f41ea1195619dfb/platforms/defaults.go#L31) also in arm.

Reply:
Correct me if I'm wrong but won't this fail? when creating it must match `linux/arm/v7` for example, whereas the architecture from the cpu variant will be something like `armv7`.

`docker buildx create --platform ${PLATFORMS} --name container-builder --use;`

Reply:
local should work unless you're trying to override the target OS/ARCH you want to compile.

Reply:
we are using `local`, only if need to load the image. Loading will work only for images that have the same platform as the host. Otherwise buildx will throw an error.

If we want to override the target OS/ARCH and its different from host, then the image will have to be pushed to a registry. right?

Reply:
Yep `local` will be sufficient here :+1: